assets/styles/app.scss

```diff
@@ -26,6 +26,19 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
 @import "~plyr/src/sass/plyr.scss";
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;900&family=Staatliches&display=swap');
 
+:root {
+
+    --plyr-color-main: #581EA3;
+
+    --plyr-range-thumb-background: #ffffff;
+
+    --plyr-control-icon-size: 20px;
+
+    --plyr-video-control-color: #ffffff;
+
+    --plyr-video-control-color-hover: #FFE500;
+}
```

Comment:
Unless Plyr makes you declare variables like that, you should use Sass syntax like those in lines 1 to 20.
Otherwise, ignore this comment.

Reply:
You can't modify the css of the player with variables like $primary

Reply:
Ok